### ISSUE TITLE
tritondigital in host list blocks NPR's podcasts

### ISSUE THREAD

**Issue by nowakommunikation** [StevenBlack/hosts#839]
Hi Steven,

I'm a Pi Hole user and your list was one of the first I added. Recently I had troubles downloading NPR podcasts via iTunes. I wrecked my brain what the issue could be and shortly before re-installing my iPod it dawned on me that I should check the Pi Hole query log. And lo and behold, the tritondigital website(s) were blocked by your list. Either you added them recently or NPR has switched to their service to provide their podcasts. 
Any reason for adding them? I would imagine that many other users also have troubles since with using iTunes. I suggest removing that domain and its subdomains. Since I unticked your list, downloading works fine.

Cheers,
Boris

**Comment by lightswitch05** [comment]
Looks like this one is my fault. I just removed all the tritondigital hosts in commit https://github.com/lightswitch05/hosts/commit/293da652c1e7985a9dbac9e1e94a28d0821a999b - it will be included in the next list update

**Comment by StevenBlack** [comment]
Thanks @nowakommunikation and thanks Daniel @lightswitch05!

Closing; this will be live and online here shortly.

**Comment by DomPizzie** [comment]
Was just about to open a new issue and saw this one. To be specific, the URL "npr.mc.tritondigital.com" is the one causing issues downloading podcast (PocketCasts in my case). I don't know about the reputation of the domain itself and/or what affect it has.

**Comment by StevenBlack** [comment]
Dominique @DomPizzie there is no longer any `tritondigital.com` domain in this list, at all.
